src/applications/edu-benefits/complaint-tool/containers/IntroductionPage.jsx

```diff
@@ -56,6 +58,9 @@ class IntroductionPage extends React.Component {
           startText="Submit Your Feedback"
           {...this.props.saveInProgressActions}
           {...this.props.saveInProgress}/>
+        <div className="omb-info--container" style={{ paddingLeft: '0px' }}>
```

Comment:
do we have to do this padding override a lot?

Reply:
We should create a TODO item to create a class for this, good point. I will create that in a subsequent PR.